docs/sphinx/user_guide/feature/policies.rst

```diff
@@ -781,14 +778,12 @@ cuda/hip_atomic_explicit      any CUDA/HIP  Atomic operation performed in a CUDA
                                             argument. See additional explanation
                                             and example below.
 builtin_atomic                seq_exec,     Compiler *builtin* atomic operation.
-                              loop_exec,
                               any OpenMP
                               policy
-auto_atomic                   seq_exec,     Atomic operation *compatible* with loop
-                              loop_exec,    execution policy. See example below.
-                              any OpenMP    Can not be used inside cuda/hip
-                              policy,       explicit atomic policies.
-                              any
+auto_atomic                   seq_exec,     Atomic operation *compatible* with 
+                              any OpenMP    loop execution policy. See example 
```

Comment:
Maybe this might be more clear?

```suggestion
auto_atomic                   seq_exec,     Atomic operation *compatible* with 
                              any OpenMP    most execution policies. See example 
```

Reply:
It works will ALL loop execution policies. https://github.com/LLNL/RAJA/blob/develop/include/RAJA/policy/atomic_auto.hpp#L41